examples/step-70/parameters.prm

```diff
@@ -8,7 +10,7 @@ subsection Stokes Immersed Problem
   set Number of time steps               = 501
   set Velocity degree                    = 2
   set Viscosity                          = 1
-  set Output directory                   = results
+  set Output directory                   = .
```

Comment:
It would probably be good practice to check whether the directory exists and, if not, create it. (Or just unconditionally create it with `mkdir -p`.)

Reply:
Is it easy to do in a cross OS compatible way? Does `mkdir -p` work on windows?

Reply:
We use this in ASPECT, which I think should be portable:
```
    int
    mkdirp(std::string pathname,const mode_t mode)
    {
      // force trailing / so we can handle everything in loop
      if (pathname[pathname.size()-1] != '/')
        {
          pathname += '/';
        }

      size_t pre = 0;
      size_t pos;

      while ((pos = pathname.find_first_of('/',pre)) != std::string::npos)
        {
          const std::string subdir = pathname.substr(0,pos++);
          pre = pos;

          // if leading '/', first string is 0 length
          if (subdir.size() == 0)
            continue;

          int mkdir_return_value;
          if ((mkdir_return_value = mkdir(subdir.c_str(),mode)) && (errno != EEXIST))
            return mkdir_return_value;

        }

      return 0;
    }
```

I don't think this has to happen for 9.2, but it would probably be useful in the long run.

Reply:
let's do this after 9.2 please.